--- a/deps/wxWidgets/wxWidgets.cmake	2026-07-28 00:37:18.712947495 +0300
+++ b/deps/wxWidgets/wxWidgets.cmake	2026-07-28 00:38:08.750428472 +0300
@@ -27,6 +27,7 @@
     GIT_REPOSITORY "https://github.com/SoftFever/Orca-deps-wxWidgets"
     GIT_SHALLOW ON
     DEPENDS ${PNG_PKG} ${ZLIB_PKG} ${EXPAT_PKG} ${JPEG_PKG}
+    PATCH_COMMAND ${CMAKE_CURRENT_LIST_DIR}/../fix-cmake4.sh <SOURCE_DIR>
     FORCE_RELEASE_CONFIG  # wxWidgets doesn't support RelWithDebInfo configuration
     CMAKE_ARGS
         -DwxBUILD_PRECOMP=ON
